src/many-modules/src/_4_events.rs

```diff
@@ -962,7 +960,8 @@ mod test {
                                 threshold: None,
                                 timeout_in_secs: None,
                                 execute_automatically: None,
-                                data: None,
+                                data_: None,
+                                memo_: None,
```

Comment:
nit: Impl sane `defaults()`

Reply:
I don't want to be gloomy, but `default()` is how block 9400 came to be :/ I'm tempted to remove it from everywhere until we have a solid solution.